packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -1783,6 +1789,7 @@ export class ProjectService extends BaseService {
         metricQuery: MetricQuery;
         queryTags: RunQueryTags;
         invalidateCache?: boolean;
+        forceUseCache?: boolean;
```

Comment:
Cache is a opt-in feature and should only be controlled/enabled via the feature flag.